src/components/OnLoadPopUp.jsx

```diff
@@ -3,19 +3,36 @@ import Dialog from "@mui/material/Dialog";
 import DialogContent from "@mui/material/DialogContent";
 import DialogContentText from "@mui/material/DialogContentText";
 import DialogTitle from "@mui/material/DialogTitle";
+import { Button, DialogActions } from "@mui/material";
+import useLocalStorage, { deleteFromStorage, writeStorage } from "@rehooks/local-storage";
 
-export default function OnLoadPopUp() {
-  const [open, setOpen] = React.useState(true);
+//this is the key for the current rev of the popup.  If you have something else you want to show,
+// e.g. you want a popup announcing an epic, change this key announce an epic, change ths key
+const SHOW_KEY_NAME = "showPopup0";
+const handleClose = () => {
+  //write false to local storage, so this browser doesnt see it again
+  writeStorage(SHOW_KEY_NAME, false);
+  //this is an optional cleanup loop just to be a good citizen.  it destroys ALL localStrage entries that are not
+  // our key, so we do not store an increasing number of keys in the browser.  Practically speaking, triden will not
+  // be storing enough keys to cause anybody problems, so you can omit if extra keys does not bother you.
+  // if you start to use localStorage elsewhere, you will need o make this more precise to delete only
+  // popup releated keys
+  Object.entries(localStorage).forEach(([key]) => {
+    if (key !== SHOW_KEY_NAME) {
+      deleteFromStorage(key);
```

Comment:
I'd regex match this for older showPopup instances, or this could be a landmine for any other localstorage we want to do later on.

Reply:
Yes, as noted in the comment directly above that line :)